.evergreen/csfle/activate_venv.sh

```diff
@@ -13,11 +13,14 @@ fi
 # create venv on first run
 if [ ! -d kmstlsvenv ]; then
    FIRST_RUN=1
-   virtualenv -p ${PYTHON_BINARY} kmstlsvenv
+   ${PYTHON_BINARY} -m venv kmstlsvenv
 fi
 
 # always activate venv
 if [ "Windows_NT" = "$OS" ]; then
+  # Workaround https://bugs.python.org/issue32451:
+  # kmstlsvenv/Scripts/activate: line 3: $'\r': command not found
+  dos2unix kmstlsvenv/Scripts/activate || true
```

Comment:
We use this same approach in pymongo's test suite here: https://github.com/mongodb/mongo-python-driver/blob/75685c006c9184230af131a09611aa3e6e1ac649/.evergreen/utils.sh#L23-L25